core/templates/dev/head/services/assets-backend-api.service.ts

```diff
@@ -198,9 +202,16 @@ angular.module('oppia').factory('AssetsBackendApiService', [
     };
 
     var _getDownloadUrl = function(entityType, entityId, filename, assetType) {
+      var urlTemplate = null;
+      if (assetType === ASSET_TYPE_AUDIO) {
+        urlTemplate = AUDIO_DOWNLOAD_URL_TEMPLATE;
+      } else if (assetType === ASSET_TYPE_THUMBNAIL) {
+        urlTemplate = THUMBNAIL_DOWNLOAD_URL_TEMPLATE;
+      } else {
+        urlTemplate = IMAGE_DOWNLOAD_URL_TEMPLATE;
```

Comment:
Should we use a dict ASSET_TYPE_TO_DOWNLAOD_URL_TEMPLATE?

(Considering I'm going to add another assetType for voiceover applications.)

Reply:
Done.